jekyll/_cci2/config-editor.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: classic-docs
+title: "Using the CircleCI In-app Configuration Editor"
+description: "Docs page on In-app configuration editor use and features"
+version:
+- Cloud
+- Server v2.x
+---
+
+A UI environment, the CircleCI config editor, is provided for users who wish to modify their configurations without the use of the
```

Comment:
I don't think user's can use the local-cli to make changes to their configuration so much as just editing it in the text-editor?

Reply:
@teesloane you could use the clif or validation, but yeah you're right it'd be more like comparing the use of the cli/an editor vs the config editor where you get validation and editing in one screen. I added next to this -- "and/or a text editor. Using the CircleCI configuration editor eases the developer experience by giving users the ability to modify their CI/CD processes in a more unified fashion."

wdyt?

Reply:
Sounds good to me!